apis/openstack-swift/src/main/java/org/jclouds/openstack/swift/v1/blobstore/RegionScopedSwiftBlobStore.java

```diff
@@ -476,7 +476,12 @@ public String completeMultipartUpload(MultipartUpload mpu, List<MultipartPart> p
    @Override
    public MultipartPart uploadMultipartPart(MultipartUpload mpu, int partNumber, Payload payload) {
       String partName = getMPUPartName(mpu, partNumber);
+
       String eTag = api.getObjectApi(regionId, mpu.containerName()).put(partName, payload);
+      if (eTag == null) {
```

Comment:
Sorry I do not understand this.  Why is the ETag nullable for a successful put?  Otherwise we should raise an exception?